### DISCUSSION TITLE
besoin d'aide fooocus/colabs pro

### DISCUSSION THREAD

**Discussion by DBMC0613** [community/community#143578] [Discussions]
### Select Topic Area

Question

### Body

Bonjour, j'espère que vous allez bien, je vous sollicite car j'ai besoin d'aide.
je n'arrive plus a accéder au fonction de fooocus avec mon abonnement google colabs pro 
Avant que ça ne marche plus (édition d'image) je suis allé dans paramètres puis dans l'onglet préréglage: pour basculer de initial a réaliste, de la plus rien n'a fonctionné, c'est a dire aucune visibilité des aperçues et toutes demande de changement de style non acceptés, fooocus se lance mais une fois dessus je ne peux travailler  
ps: je ne sais absolument pas écrire de code et ni comprend pas grand chose 
je vous remercie

**Comment by LiteBrite82** [comment]
Hi @DBMC0613 , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
